dcm4che-imageio-opencv/src/main/java/org/dcm4che3/opencv/StreamSegment.java

```diff
@@ -157,28 +151,22 @@ private static StreamSegment getFileStreamSegment(SegmentedInputImageStream iis)
                     }
                 }
                 LOGGER.error("Cannot read SegmentedInputImageStream with {} ", fstream.getClass());
-            }
         } catch (Exception e) {
             LOGGER.error("Building FileStreamSegment from SegmentedInputImageStream", e);
         }
         return null;
     }
 
-    private static long[][] getSegments(SegmentedInputImageStream iis, Class<? extends ImageInputStream> clazz, Field fCurSegment) throws Exception {
-        Integer curSegment = (Integer) fCurSegment.get(iis);
+    private static long[][] getSegments(SegmentedInputImageStream iis) throws Exception {
```

Comment:
Do we still need to throw the generic "Exception" here? Can we make it more specific?